executor/src/witgen/jit/block_machine_processor.rs

```diff
@@ -520,19 +520,19 @@ main_binary::operation_id_next[0] = main_binary::operation_id[1];
 call_var(9, 0, 0) = main_binary::operation_id_next[0];
 main_binary::operation_id_next[1] = main_binary::operation_id[2];
 call_var(9, 1, 0) = main_binary::operation_id_next[1];
-2**24 * main_binary::A_byte[2] + 2**0 * main_binary::A[2] := main_binary::A[3];
+2**0 * main_binary::A[2] + 2**24 * main_binary::A_byte[2] := main_binary::A[3];
```

Comment:
These are just reordering. Is this ordering better? Could we change the code to maintain the original ordering?

Reply:
I added code to sort by exponent. Before, it it was sorted by the order it appears in the constraint. Note that when we execute such a bit decomposition, we will sort it by exponent as well, so I think it's better to sort it like this already when creating the effect.